velox/duckdb/conversion/DuckWrapper.cpp

```diff
@@ -30,6 +30,34 @@ using ::duckdb::LogicalTypeId;
 using ::duckdb::PhysicalType;
 using ::duckdb::QueryResult;
 
+namespace {
+
+class DuckDBBufferReleaser {
+ public:
+  DuckDBBufferReleaser(::duckdb::buffer_ptr<::duckdb::VectorBuffer> buffer)
+      : buffer_(std::move(buffer)) {}
+
+  void addRef() const {}
+  void release() const {}
+
+ private:
+  ::duckdb::buffer_ptr<::duckdb::VectorBuffer> buffer_;
```

Comment:
can `buffer_` and `validity_` below be `const`?

Reply:
Fixed